lib/Utils.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright 2019 The Clspv Authors. All rights reserved.
```

Comment:
I would have called this CallGraphOrderedFunctions.cpp

"util" and "misc" generally become the attic of a project, with poor cohesion.

Reply:
Good point, I'll make this change.